packages/expo-blur/src/BlurView.web.tsx

```diff
@@ -1,28 +1,51 @@
 import PropTypes from 'prop-types';
 import * as React from 'react';
-import { View, StyleSheet, ViewPropTypes } from 'react-native';
+import { View, ViewPropTypes } from 'react-native';
 
-type Props = {
-  tint: BlurTint;
-} & React.ComponentProps<typeof View>;
-type BlurTint = 'light' | 'dark' | 'default';
+import { BlurTint, Props } from './BlurView.types';
+import getBackgroundColor from './getBackgroundColor';
 
+function isBlurSupported(): boolean {
+  // https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#Browser_compatibility
+  // TODO: Bacon: Chrome blur seems broken natively
+  return CSS.supports('-webkit-backdrop-filter', 'blur(1px)');
```

Comment:
some browsers don't support CSS.supports:
```suggestion
  return typeof CSS !== 'undefined' && CSS.supports('-webkit-backdrop-filter', 'blur(1px)');
```